kodilla-testing/src/main/java/com/kodilla/testing/calculator/Calculator.java

```diff
@@ -0,0 +1,12 @@
+package com.kodilla.testing.calculator;
+
+public class Calculator {
+
+    public double add(int a, int b) {
```

Comment:
czemu zwracasz double, kiedy parametry są intami? Czy suma liczb całkowitych jest liczbą zmiennoprzecinkową?

Reply:
Uważam, że tu ogólnie powinny być double'e przyjmowane jako argumenty. Ale właściwie skoro miały być int'y to i metoda powinna zwracać int'y.